oneshot.d/browser-update-default-data

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+MOZEMBED_DIR=/home/$USER/.mozilla/mozembed
+BROWSER_DATA_DIR=/usr/share/sailfish-browser/data
+
+if [ ! -d "$MOZEMBED_DIR" ]; then
+    mkdir -p $MOZEMBED_DIR
+fi
+
+if [ ! -f "$MOZEMBED_DIR/ua-update.json" ]; then
```

Comment:
It also means that it's easier for community to test different sites and for us easier way to apply fixes.

Personally I'd like to see the next level where you guys (community) would start providing better UAs for you favourite web sites. Enablers starts to be in place :). A topic that we comment for the community a meeting? Do it together still holds. I just need to describe how to test user-agents and tell where we are going to. My thinking is that we should flip to super simple UA like B2G has. I feel that this would be beneficial to both sides.